src/suggestion.rs

```diff
@@ -113,23 +324,75 @@ impl<'s> fmt::Display for Suggestion<'s> {
             ))
             .fmt(formatter)?;
 
-        writeln!(formatter, " {}", self.chunk.as_str())?;
-
         // underline the relevant part with ^^^^^
-        let marker_size = if self.span.end.line == self.span.start.line {
-            // column bounds are inclusive, so for a correct length we need to add + 1
-            self.span.end.column.saturating_sub(self.span.start.column) + 1
-        } else {
+
+        // @todo this needs some more thought once multiline comments pop up
+        let marker_size = self.span.one_line_len().unwrap_or_else(|| {
             self.chunk
-                .as_str()
-                .chars()
-                .count()
+                .len_in_chars()
                 .saturating_sub(self.span.start.column)
+        });
+
+        // assumes the _mistake_ is within one line
+        // if not we chop it down to the first line
```

Comment:
why do you use first line here? should it be the line where errors appears?

Reply:
Currently we can only print single lines.

So what we want to do is - again - a coverage problem. If the mistakes only covers one line, this will yield the one and only line in which the mistake exists, if the mistake covers multiple lines (which is currently not possible with at least the hunspell backend) then we use the first line in which part of the mistake occurs.